content/metadata.yaml

```diff
@@ -70,6 +70,14 @@ authors:
     affiliations:
       - Department of Pathology, Perelman School of Medicine University of Pennsylvania, Philadelphia PA, USA
     funders: GBMF4552
+  - github: naglem
+    name: Michael W. Nagle
+    initials: MWN
+    orcid: 0000-0002-4677-7582
+    twitter: mikenagle84
+    email: mike_nagle@eisai.com
+    affiliations:
+      - Neurogenomics, Translational Sciences, Neurology Business Group, Eisai Inc, Cambridge, Massachusetts, United States of America
```

Comment:
Should we also add your Pfizer affiliation that was active during this research?

Reply:
Yeah, would probably be appropriate. I can modify and resend the request